internal/common/resource.go

```diff
@@ -161,6 +167,11 @@ func (r *reconcileBuilder) Options(options ReconcileOptions) ReconcileBuilder {
 	return r
 }
 
+func (r *reconcileBuilder) IgnoreObjectOwnedByVirtOperator() ReconcileBuilder {
```

Comment:
FWIW if folks think we might need to ignore other types of resource in the future I can make this more generic passing a function as with the `UpdateFunc` and `StatusFunc` methods but at the moment I think it's better to be explicit.

Reply:
IMHO this is too specific to be in the `reconcileBuilder`. Can you move the logic to the functions that reconcile `InstanceTypes` and `Preferences`? `reconcileVirtualMachineClusterInstancetypesFuncs()` and `reconcileVirtualMachineClusterPreferencesFuncs()`

There you can check if the resource is owned by virt-operator or not.